### ISSUE TITLE
Angular HttpClient module dosent allow to send date as header

### ISSUE THREAD

**Issue by shasaxena-equinix** [jessety/simple-hmac-auth#10]
Beneath $http it uses XmlHttpRequest and XmlHttpRequest isn't allowed to set the Date header as per the standard.

Step 5 states:

Terminate these steps if header is a case-insensitive match for one of the following headers...

and Date is included in that list.

find more details [here ](https://www.w3.org/TR/XMLHttpRequest/#the-setrequestheader()-method)

I have made changes in the code and changed date gheader to something like requestdate, this works totally fine for me.

I would request you to change date header with some other meaning full word.

**Comment by drone1** [comment]
The [other active issue in this repo](https://github.com/jessety/simple-hmac-auth/issues/7) is one I posted about. Same issue. I don't understand how anyone is able to use this package in any modern browser, since the ``date`` header is not allowed. I wish the author (Hi, @jessety) would reply to one of us? I forked ages ago and simply renamed the header to ``timestamp`` and it worked fine, but I'd love to be using this repo rather than trying to maintain my own. I sure would appreciate it if @jessety could take a few moments to reply here?
